src/authentication/azure/Changelog-old.md

```diff
@@ -0,0 +1,130 @@
+# Changelog (old)
+
+All notable changes to this project will be documented in this file.
```

Comment:
for all the implementation libraries I think keeping those should be ok since the releases are on other repos.